3.0/vector_tile.proto

```diff
@@ -24,7 +24,18 @@ message Tile {
                 optional sint64 sint_value = 6;
                 optional bool bool_value = 7;
 
-                extensions 8 to max;
+                // These changes should be moved to the 3.0 dir, but leaving it here for now
+                // to make it easier to compare with the previous version
+
+                // A list of key-value pairs. Just like Feature's "tags", the repeated pairs
+                // of integers point to Layer's "keys" (1st int) and "values" (2nd int) array.
```

Comment:
Can we change "1st int" to "odd integers" and "2nd int" to "even integers"?

We should add a note that it's invalid to specify indices that are greater than or equal to the currently parsed index to explicitly disallow circular structures. Parsers are going to implement it that way in order to prevent circular structures.

Reply:
@kkaefer I fixed the odd/even (sorry i missed your comments earlier). Where/how would you like the note to be added?